src/app/static/src/tests/components/plots/bubble/utils.test.ts

```diff
@@ -220,4 +220,29 @@ describe("Bubble plot utils", () => {
         });
     });
 
+    it("it compute prevalence and plhiv uncertainty where ranges are zeros", () => {
```

Comment:
Great, this is now testing that the features come out of the indicators as expected. Could you also add a test that the tooltip is generated correctly?